app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>Homepages#index</h1>
+<p>Find me in app/views/homepages/index.html.erb</p>
+
+<% if Work.find_highest != nil %>
+<% high = Work.find_highest%>
+<h2>Media Spotlight: <%= link_to "#{high.title}", work_path(high.id) %> by <% high.creator %> </h2>
+<p> <%= Vote.where(work_id: high.id).count %> votes - <%= high.description %></p>
+<% end %>
+
+<h2 class="top-ten__header">Top Movies</h2>
+
+<% Work.list_works("movie")[0,10].each do |movie| %>
+<h4></h4><%= movie.title %></h4>
+<p>by <%= movie.creator %></p>
+<p><%= Vote.where(work_id: movie.id).count %> votes</p>
+<% end %>
+
+<h2 class="top-ten__header">Top Books</h2>
+<% Work.list_works("book")[0,10].each do |book| %>
+<h4><%= book.title %></h4>
+<p>by <%= book.creator %></p>
+<p><%= Vote.where(work_id: book.id).count %> votes</p>
+<% end %>
+
+<h2 class="top-ten__header">Top Albums</h2>
+
+<% Work.list_works("album")[0,10].each do |album| %>
```

Comment:
I suggest limiting the works returned to 10, in the `list_works` method, or do something else to make it paginate.